src/cilrep.ml

```diff
@@ -3498,8 +3497,7 @@ let _ =
             if !Errormsg.hadErrors then
               Errormsg.parse_error
                 "cilRep: fill_va_table: failure while preprocessing stdio header file declarations\n";
-            iterGlobals cilfile (fun g ->
-                match g with
+            iterGlobals cilfile (function
```

Comment:
@jlacomis another pattern messes up here because g is used outside the `match` expression.

Reply:
Oh, interesting. Hadn't thought of that one.